core/src/main/java/com/datastax/oss/driver/internal/core/context/DefaultNettyOptions.java

```diff
@@ -74,6 +77,24 @@ public DefaultNettyOptions(InternalDriverContext context) {
             .setNameFormat(context.getSessionName() + "-admin-%d")
             .build();
     this.adminEventLoopGroup = new DefaultEventLoopGroup(adminGroupSize, adminThreadFactory);
+    // setup the Timer
+    ThreadFactory timerThreadFactory =
+        new ThreadFactoryBuilder().setThreadFactory(safeFactory).build();
```

Comment:
Could you use a name format like the two other factories? This is pretty convenient when debugging stuff (thread dumps, etc.)
```suggestion
        new ThreadFactoryBuilder().setThreadFactory(safeFactory).setNameFormat(context.getSessionName() + "-timer-%d").build();
```